products/memgraph/upcoming/tutorials/got-deaths.md

```diff
@@ -0,0 +1,232 @@
+## Game of Thrones deaths
+
+This article is a part of a series intended to show how to use Memgraph
+on real-world data to retrieve some interesting and useful
+information.
+
+We highly recommend checking out the other articles from this series which
+are listed in our [tutorial overview section](tutorials-overview.md).
+
+### Introduction
+
+**WARNING** - If you've been living under a rock and still have not watched Game of Thrones,
+this tutorial could contain ***spoilers***. 
+
+We can all agree that 2020. was one of the worst years in our lives because of the worldwide
+pandemic, 2019. still comes close with that disappointing, poorly written ending to one of the
+best shows of all time, Game of Thrones. Seven-year build-up to that awful last season crushed
+the souls of millions of fans around the world. Nevertheless, the Game of Thrones world is full
+of interesting characters, locations, scenarios, and most importantly deaths. If you want to know
+how many characters would survive if Jon Snow stayed dead, which House has the best Kill/Death Ratio
+or who was the biggest traitor in the show, you have come to the right place!
+
+### Data Model
+
+Although you know the Game of Thrones TV show is based on series of books called A Song of Ice and

```

Comment:
Try to skip `you know` when doing tech documentation except when referencing material in paragraph like `As you probably notes in the previous example`, `You could see that this code throws a new error`. Try to skip general knowledge "you know"s.